cookbooks/aws-parallelcluster-config/templates/default/directory_service/sssd.conf.erb

```diff
@@ -0,0 +1,38 @@
+[domain/default]
+debug_level = 0x1f0
+id_provider = ldap
+cache_credentials = True
+ldap_schema = AD
+ldap_uri = <%=  node['cluster']["directory_service"]["domain_addr"] %> # TODO: use ldaps
+ldap_search_base = <%=  node['cluster']["directory_service"]["domain_name"] %>
+ldap_default_bind_dn = <%=  node['cluster']["directory_service"]["domain_read_only_user"] %> # TODO: parametrize username
+ldap_default_authtok_type = obfuscated_password
+ldap_default_authtok = <%= @obfuscated_password %>
+ldap_tls_cacert = /etc/openldap/certs/privatekey.pem
+ldap_tls_reqcert = never # TODO: use 'require'
+ldap_id_mapping = True
+ldap_referrals = False
+enumerate = True
+fallback_homedir = /home/%u
+default_shell = /bin/bash
+ldap_auth_disable_tls_never_use_in_production = 'true'  # TODO:  use SSL
+use_fully_qualified_names = False
```

Comment:
let's take a TODO to figure out which of these we need to set (or don't necessarily need to, but want to anyways, like fallback_homedir and default_shell).

enumerate we should leave out i think, since it can lead to lots of time spent in enumerating a directory for large domains

Reply:
Added a todo